libarchive/CMakeLists.txt

```diff
@@ -252,6 +252,7 @@ ENDIF(BUILD_SHARED_LIBS)
 
 # archive_static is a static library
 ADD_LIBRARY(archive_static STATIC ${libarchive_SOURCES} ${include_HEADERS})
+TARGET_INCLUDE_DIRECTORIES(archive_static PUBLIC .)
```

Comment:
Okay.  We can take this.

I would prefer to only advertise the actual public header files, but that would require reorganizing the sources a bit.  If you're interested in doing that, please let me know and I can help.